gradle.properties

```diff
@@ -1,11 +1,11 @@
 projectVersion=1.1.1.BUILD-SNAPSHOT
-micronautVersion=1.1.3
+micronautVersion=1.2.7
```

Comment:
Is it really necessary to upgrade the Micronaut version? If it works with an older version as a dependency, then it's better so we don't "force" users to upgrade.